tensorflow/lite/experimental/delegates/coreml/coreml_delegate.h

```diff
@@ -15,11 +15,12 @@ limitations under the License.
 #ifndef TENSORFLOW_LITE_EXPERIMENTAL_DELEGATES_COREML_COREML_DELEGATE_H_
 #define TENSORFLOW_LITE_EXPERIMENTAL_DELEGATES_COREML_COREML_DELEGATE_H_
 
-#include "tensorflow/lite/c/common.h"
```

Comment:
what's the benefit of removing this?

Reply:
The benefit is faster compile time and reduced binary size besides avoiding namespace pollution.

Reply:
I don't see the improvement on my side. What's your test method?